recover/recovery_test.go

```diff
@@ -162,7 +154,7 @@ func TestRecoverer_RecoverFromXMRMakerSecretAndContract_Claim_afterTimeout(t *te
 	require.NoError(t, err)
 
 	claimKey := keys.Secp256k1PublicKey.Keccak256()
-	addr, contract, swapID, swap := newSwap(t, claimKey, [32]byte{}, true)
```

Comment:
This puts the logic back to the way it was before my previous PR. The `setReady` value here is the only thing that differentiates this test from the previous test.